src/webrtc/callFeed.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import EventEmitter from "events";
+
+export enum CallFeedPurpose {
+    Usermedia = "usermedia",
+    Screenshare = "screenshare",
+}
```

Comment:
It would make sense to put these into `callEventTypes.ts` as `SDPStreamMetadataPurpose` to match MSC3077